cicd/build.sh

```diff
@@ -16,21 +16,37 @@ set -e
 
 source ${CICD_ROOT}/_common_container_logic.sh
 
+is_pr_or_mr_build() {
+    [ -n "$ghprbPullId" ] || [ -n "$gitlabMergeRequestId" ]
+}
+
+get_expiry_label_parameter() {
+    echo "--label quay.expires-after=${QUAY_EXPIRE_TIME}"
+}
+
+is_rhel7_host() {
+
+    local RELEASE_FILE='/etc/redhat-release'
+
+    [ -f "$RELEASE_FILE" ] && grep -q -i "release 7" "$RELEASE_FILE"
+}
+
 function build {
+
+    local IMAGE_TAG_LATEST=''
```

Comment:
This starts to break from the pattern of ENV variables being 'global' to the bonfire cicd scripts, where everything is exported in bootstrap.

`IMAGE_TAG_LATEST` is indeed currently only used in this function, so its currently a safe move to local, but I think it would be better to maintain the consistency of exporting that variable. Going as far as defining it in `bootstrap.sh`.

Reply:
Approving with this optional input, as the variable currently isn't exporting so I'm asking you to expand the scope of your intent.

Reply:
I don't think IMAGE_TAG_LATEST was meant to be used outside of this func so I'm ok with it staying local.

Reply:
I hear you Mike, but as Brandon confirmed, I examined the code and IMAGE_TAG_LATEST is not used anywhere, nor meant to be used to configure tags for the image at top level, so that's why I refactored it to stay as local and prevent it from going to the external scope needlessly, hope you all agree.